### DISCUSSION TITLE
sync fail

### DISCUSSION THREAD

**Discussion by sahandileep** [WhatsApp/stickers#762] [Q&A] [answered]
Gradle sync failed: Cannot query the value of this provider because it has no value available.
					Consult IDE log for more details (Help | Show Log) (2 s 65 ms)

A problem occurred configuring project ':app'.
> Cannot query the value of this provider because it has no value available.

License for package Android SDK Build-Tools 29.0.2 not accepted.

when sync is started it fails with errors please help me

**Comment by simonzhexu** [comment, marked as answer]
License for package Android SDK Build-Tools 29.0.2 not accepted. 

This seems to be the problem.

**Reply by sahandileep** [reply]
Thanks for responding. Could you help me to solve this? Please let me know.

**Reply by simonzhexu** [reply]
Go to SDK manager, and download another SDK, the license will come up, just accept it.

**Reply by sahandileep** [reply]
hello friend, Could you help to fix this error

Build file 'C:\Users\sahan\Desktop\New folder (2)\stickers-master\stickers-master\Android\app\build.gradle' line: 73

Execution failed for task ':app:checkDebug'.
> Your applicationId is currently com.example.samplestickerapp, please change your applicationId to a different string in app/build.gradle in line 10
